client/scss/components/_button.scss

```diff
@@ -330,6 +330,16 @@
     }
   }
 
+  // For buttons we want to look like inline links
+  &.plain {
```

Comment:
Should we add these to the pattern library and styleguide template?

Note: We actually recently removed something similar `unbutton` as it was not used, not sure if we should revisit this class that was already present in the code instead of adding a new name?

Alternatively, should we name this to be a bit clearer - `.as-link` or `w-button--as-link` (for longer term adoption of BEM classes). Or does this get close enough if we add `.link` to the button?

- see https://github.com/wagtail/wagtail/pull/9044 / 1d5824dcc796074ddfb98ab763646c83bba24e3f
- done as part of https://github.com/wagtail/wagtail/issues/8790
-